### DISCUSSION TITLE
New Addition for Website Monitoring Cat

### DISCUSSION THREAD

**Discussion by GraeChris** [abishekmuthian/startuptoolchain#10] [Ideas]
An open source alternative to Uptime Robot that is self-hosted. It's deployable via a docker container. I would suggest not deploying on Azure as Microsoft intentionally blocks outbound ICMP and this severely handicaps the software. 

https://github.com/louislam/uptime-kuma

**Comment by abishekmuthian** [comment]
Thank you for your suggestion, I have added Uptime Kuma to Startup Toolchain!
